client/displayservers/Wayland/wayland.c

```diff
@@ -47,6 +50,20 @@ Place, Suite 330, Boston, MA 02111-1307 USA
 #include "wayland-relative-pointer-unstable-v1-client-protocol.h"
 #include "wayland-idle-inhibit-unstable-v1-client-protocol.h"
 
+#define EPOLL_EVENTS 10 // Maximum number of fds we can process at once in waylandWait
+
+typedef void (*WaylandPollCallback)(uint32_t events, void * opaque);
+
+struct WaylandPoll
+{
+  int fd;
+  bool removed;
+  WaylandPollCallback callback;
+  void * opaque;
+  struct WaylandPoll * prev;
```

Comment:
Use `struct wl_list link;` instead.

https://wayland.freedesktop.org/docs/html/apb.html

Reply:
Switched to `wl_list`.